app/components/search-preprints.js

```diff
@@ -1,4 +1,8 @@
-import Ember from 'ember';
+import { inject } from '@ember/service';
+import $ from 'jquery';
```

Comment:
Is this necessary? I think `$` is just globally available

Reply:
This ensures you're using the bundled jQuery